### DISCUSSION TITLE
Add hostname in the Endpoint configuration

### DISCUSSION THREAD

**Discussion by lucascanna** [mia-platform/community#160] [Ideas]
Hi! 

It would be nice to have the possibility to expose services, CRUD etc.. under different hostnames.

In particular, in the endpoints section, I would like to have the possibility to choose the hostname (not only the path). The console should let you choose between the default hostname associated to the project or creating a new one. 

Of course, this implies that the console should automatically generate (and deploy) the Kubernetes ingress resources.

What do you guys think about it?

**Comment by davidebianchi** [comment]
Interesting ideas. We already are thinking about the integration of the ingress resources inside the resources supported by the Console. After this, we could also add this feature.

**Comment by akelity** [comment]
This feature has been positively evaluated and may be considered for inclusion in the roadmap in the future, depending on priorities and overall specific needs.
